release/nightly_tests/dataset/read_and_consume_benchmark.py

```diff
@@ -58,6 +63,11 @@ def consume_fn(ds):
             for _ in ds.iter_internal_ref_bundles():
                 pass
 
+    elif args.write:
+
+        def consume_fn(ds):
+            ds.write_parquet(WRITE_PATH)
```

Comment:
Do we add a delete to cleanout these S3 objects written out? Don't want to accumulate a lot of data here.

Reply:
Yeah, I think they should automatically get cleaned out, but I'll monitor the bucket to confirm that the lifecycle policy works as expected